src/test/java/seedu/address/logic/commands/SellCommandTest.java

```diff
@@ -99,6 +98,15 @@ public void execute_sellNonExistentGood_throwCommandException() throws CommandEx
                 SellCommand.MESSAGE_SELLING_NONEXISTENT_GOOD, () -> sellCommand.execute(modelStub));
     }
 
+    @Test
+    public void execute_validPerson_callsModelCommit() throws CommandException {
```

Comment:
Why is there a ```validPerson``` in the name? 

Should it be something like ```execute_callsModelCommit_modelCommitSuccessful()``` instead?

Reply:
Oops, leftover from copypasta, will change to execute_validTransaction_callsModelCommit.

It should be methodName_input_result format, right?

Reply:
>>methodName_input_result format, right?
yes. The middle could be input or action.